doc/source/whatsnew/v1.0.0.rst

```diff
@@ -211,6 +211,7 @@ Other
 - Trying to set the ``display.precision``, ``display.max_rows`` or ``display.max_columns`` using :meth:`set_option` to anything but a ``None`` or a positive int will raise a ``ValueError`` (:issue:`23348`)
 - Using :meth:`DataFrame.replace` with overlapping keys in a nested dictionary will no longer raise, now matching the behavior of a flat dictionary (:issue:`27660`)
 - :meth:`DataFrame.to_csv` and :meth:`Series.to_csv` now support dicts as ``compression`` argument with key ``'method'`` being the compression method and others as additional compression options when the compression method is ``'zip'``. (:issue:`26023`)
+- Bug in :meth:`Series.diff` where a boolean series would cause a TypeError (the - operator is deprecated) when using NumPy >= 1.13.0 (:issue:`17294`)
```

Comment:
Bug in :meth:`Series.diff` where a boolean series would incorrectly raise ``TypeError`` (:issue:....)

Reply:
Quick question: if people start looking for that bug ('the - operator is deprecated'), how would they know to update pandas if I remove the second part?